templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,17 @@
+# Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button ![chevron](/static/images/help/chevron.png)
+to bring up the stream dropdown menu and press **Mute the stream**. After
```

Comment:
@540KJ You should probably add the **stream dropdown** image after this sentence. Also I would suggest you to use an image which has a highlight over **Mute the stream** like your unmute_option.png.